.gitignore

```
@@ -2,5 +2,5 @@
 *.jl.cov
 *.jl.mem
 /Manifest.toml
-/docs/Manifest.toml
+/test/Manifest.toml
```

This gitignores `test/Manifest.toml` but removes `docs/Manifest.toml` from the gitignore. You should just add a new line for `test/Manifest.toml`.

---

Actually thought some more about this - let's leave the gitignore as is. Will leave a new comment with the suggested change.

---

Actually thought some more about this - let's leave the gitignore as it was for this PR.

```suggestion
/docs/Manifest.toml
```

---

Please address!